src/search/utils/logging.h

```diff
@@ -132,10 +133,35 @@ extern void add_log_options_to_feature(plugins::Feature &feature);
 extern LogProxy get_log_from_options(const plugins::Options &options);
 extern LogProxy get_silent_log();
 
+class ContextError : public utils::Exception {
+public:
+    explicit ContextError(const std::string &msg);
+};
+
+class Context {
+protected:
+    static const int INDENT_AMOUNT = 2; // TODO: Why not directly the string?
```

Comment:
I don't know why it is the way it is. Both solutions seem fine to me.

Reply:
We switch to a string constant